docs/model_access_control.md

```diff
@@ -0,0 +1,652 @@
+# Model Access Control
+
+In the present setting, on a security enabled OpenSearch cluster, we can limit users to certain actions by mapping them to relevant permissions and permission roles. For example, we may want some users to only search and view ml models while allow others to register and deploy them. 
```

Comment:
"we can limit users..." -> "admin can limit user..."

"we may want some users... " -> "admin can grant some user access to .."